global/scripts/FieldMapPreprocessingAll.sh

```diff
@@ -42,14 +58,15 @@ opts_AddMandatory '--ofmapmagbrain' 'MagnitudeBrainOutput' 'image' "output disto
 
 opts_AddMandatory '--ofmap' 'FieldMapOutput' 'image' "output distortion corrected fieldmap image (rad/s)"
 
-#Optional Arguments
-opts_AddOptional '--echodiff' 'DeltaTE' 'number (milliseconds)' "echo time difference for Siemens and Philips fieldmap images (in milliseconds)"
+# options --echodiff is now mandatory - used by all "--method" (and all manufacturer). 
+opts_AddMandatory '--echodiff' 'DeltaTE' 'number (milliseconds)' "echo time difference for fieldmap images (in milliseconds)"
 
-opts_AddOptional '--fmap' 'GEB0InputName' 'number (degrees)' "input General Electric fieldmap with fieldmap in deg and magnitude image"
+# Optional Arguments
+opts_AddOptional '--fmap' 'GEB0InputName' 'image (hertz and magnitude)' "input GE HealthCare Legacy fieldmap with fieldmap in hertz and magnitude image"
```

Comment:
```suggestion
opts_AddOptional '--fmap' 'GEB0InputName' 'image (hertz and magnitude)' "GE HealthCare Legacy fieldmap with field map in Hz and magnitude image included as two volumes in a single file"
```

Reply:
Also, should we perhaps change the forward facing name of this flag to `--fmapgelegacy` (which including `--fmap` as an additional recognized option for backwards compatibility)?